### PR TITLE
Log setting / unsetting of contest mode

### PR DESCRIPTION
**Pull request by 13steinj**
Log the setting / unsetting of contest mode in the subreddit modlog.

Thanks to @Pokechu22 for the removal image. I added the plus sign / insignia to the other image because of my OCD.

:eyeglasses: @MelissaCole

### CODE REVIEW

File: r2/r2/controllers/api.py [hunk 1]
```diff
@@ -1574,6 +1574,16 @@ def POST_set_contest_mode(self, form, jquery, thing, state):
         """
         thing.contest_mode = state
         thing._commit()
+        if state == True:
+            action = 'setcontestmode'
+        elif state == False:
```
Comment (MelissaCole) [inline r2/r2/controllers/api.py]: I think it's better to be `if state:` and `if not state:`

File: r2/r2/controllers/api.py [hunk 2]
```diff
@@ -1574,6 +1574,16 @@ def POST_set_contest_mode(self, form, jquery, thing, state):
...
+            action = 'unsetcontestmode'
+        ModAction.create(
+            c.site,
+            c.user,
+            action,
+            target=thing
```
Comment (MelissaCole) [inline r2/r2/controllers/api.py]: I'd prefer it on one line since it fits

**Comment by MelissaCole** [comment]
:nail_care: Just some really tiny nits but looks good. Thanks!

**Comment by 13steinj** [comment]
@MelissaCole Sure thing, done. :haircut:

**Comment by MelissaCole** [comment]
Thanks! :fish:

**Comment by MelissaCole** [comment]
3338475

**Pull request closed without merging**